pom.xml

```diff
@@ -163,7 +163,7 @@
             <dependency>
                 <groupId>io.jenkins.tools.bom</groupId>
                 <artifactId>bom-${jenkins.baseline}.x</artifactId>
-                <version>3654.v237e4a_f2d8da_</version>
+                <version>3944.v1a_e4f8b_452db_</version>
```

Comment:
Do not `renovate` update this? I did not use renovate in other plugins, for several reasons, one it is complicated to configure for the simple needs we have in Jenkins plugins, dependabot is enough for that and is simpler to configure, dependabot is maintained by GitHub and integrate with other security tool Github has.

Reply:
there is no thing like dependabot or renovate on this project. Is this something we could or should add?

Reply:
I saw this https://github.com/jenkinsci/opentelemetry-api-plugin/blob/main/.github/renovate.json so I assume renovate is configured, if it is not configure, we should configure dependabot to bump dependencies automatically

Reply:
Please see https://github.com/jenkinsci/opentelemetry-api-plugin/pull/22